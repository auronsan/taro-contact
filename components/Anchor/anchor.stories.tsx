import React from 'react';
import { Anchor } from './index';

export default {
  title: 'components/Anchor',
  component: Anchor,
};

export const WithHref = () => <Anchor href="https://example.com">Click Here</Anchor>;

export const WithCustomClassName = () => (
  <Anchor className="custom-class" id="first-child" href="https://example.com">
    Click Here
  </Anchor>
);

export const WithInlineStyles = () => (
  <Anchor style={{ backgroundColor: 'red' }} id="first-child" href="https://example.com">
    Click Here
  </Anchor>
);
