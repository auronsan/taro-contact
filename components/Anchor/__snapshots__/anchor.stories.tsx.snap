// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Anchor WithCustomClassName smoke-test 1`] = `
<main class="roboto-normal">
  <a href="https://example.com"
     class="custom-class anchor_root__e1GyS"
     id="first-child"
     data-testid="first-child"
  >
    Click Here
  </a>
  <div class="stack_root__KVCbK toast_toast-container__8fZtQ stack_gap-xs__xTnny"
       id="toast-container"
       data-testid="toast-container"
       style="padding: 0px;"
  >
  </div>
</main>
`;

exports[`components/Anchor WithHref smoke-test 1`] = `
<main class="roboto-normal">
  <a href="https://example.com"
     class="anchor_root__e1GyS"
  >
    Click Here
  </a>
  <div class="stack_root__KVCbK toast_toast-container__8fZtQ stack_gap-xs__xTnny"
       id="toast-container"
       data-testid="toast-container"
       style="padding: 0px;"
  >
  </div>
</main>
`;

exports[`components/Anchor WithInlineStyles smoke-test 1`] = `
<main class="roboto-normal">
  <a href="https://example.com"
     class="anchor_root__e1GyS"
     id="first-child"
     data-testid="first-child"
     style="background-color: red;"
  >
    Click Here
  </a>
  <div class="stack_root__KVCbK toast_toast-container__8fZtQ stack_gap-xs__xTnny"
       id="toast-container"
       data-testid="toast-container"
       style="padding: 0px;"
  >
  </div>
</main>
`;
